### PR TITLE
Add docker image for testing wasm on windows

### PR DESCRIPTION
**Pull request by radekdoulik**

### CODE REVIEW

File: src/windowsservercore/2004/helix/webassembly/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-2004-helix-amd64
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-2004-helix-amd64-20200904200251-272704c
```
Comment (mthalman) [inline src/windowsservercore/2004/helix/webassembly/Dockerfile:1]: It was right the way you had it before.  You just need to add a local tag like the following:
```json
"tags": {
  "windowsservercore-2004-helix-amd64-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
  "windowsservercore-2004-helix-amd64": {
    "isLocal": true
  }
}
```

at this point: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/441/files#diff-ffa5b716b5a57837f7929dfcca4b4dfdeb97210a7fd5a12d2f1978846d6f1743R1086-R1088

File: manifest.json [hunk 2]
```diff
@@ -1084,7 +1084,22 @@
...
+          ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/windowsservercore/2004/helix/webassembly",
+              "os": "windows",
+              "osVersion": "windowsservercore-2004",
+              "tags": {
+                "windowsservercore-2004-helix-webassembly-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
```
Comment (mthalman) [inline manifest.json:1102]:
```suggestion
                "windowsservercore-2004-helix-webassembly-amd64-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
```

File: src/windowsservercore/2004/helix/webassembly/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-2004-helix-amd64
+
+# Install git
+ENV GIT_VERSION=2.31.1
+ENV GIT_INSTALLER=MinGit-${GIT_VERSION}-64-bit.zip
+
+RUN curl -SL --output %TEMP%\%GIT_INSTALLER% https://github.com/git-for-windows/git/releases/download/v2.31.1.windows.1/%GIT_INSTALLER% \
+    && powershell -Command "Expand-Archive %TEMP%\$env:GIT_INSTALLER -DestinationPath C:\git" \
```
Comment (mthalman) [inline src/windowsservercore/2004/helix/webassembly/Dockerfile:8]: A bit simpler to use tar.

```suggestion
    && tar -C C:\git -zxf %TEMP%\%GIT_INSTALLER% \
```

**Review by mthalman — APPROVED** [review]

**Comment by radical** [comment]
Can we add chrome, and chromedriver too?

**Comment by radekdoulik** [comment]
> Can we add chrome, and chromedriver too?

I can try that. Would it work in docker though?

**Comment by radical** [comment]
> > Can we add chrome, and chromedriver too?
> 
> I can try that. Would it work in docker though?

We run that headless, so it should work fine. And I just realized that we download chrome+driver during the build, and send it to helix as a payload. So, you don't need to do anything for that here.

**Pull request merged**